README.md

```diff
@@ -1,2 +1,125 @@
+ | GET      | /eholdings/titles/{title_id}                 | Get the title by title_id.                                                            |
+ | POST     | /eholdings/titles/{title_id}                 | Create a new Custom Title.                                                            |
+ | GET      | /eholdings/proxy-types                       | Get a list of supported root proxy types.                                             |
+ | GET      | /eholdings/root-proxy                        | Get a list of Root Proxy.                                                             |
+ | PUT      | /eholdings/root-proxy                        | Update the current root proxy.                                                        |
+ | GET      | /eholdings/configuration                     | Get KB configuration currently being used.                                            |
+ | PUT      | /eholdings/configuration                     | Update the currently set KB configuration.                                            |
+ | GET      | /eholdings/status                            | Retrives status of current KB configuration.                                          |
+
+## Compiling
+
+```
+   mvn install
+```
+
+See that it says "BUILD SUCCESS" near the end.
+
+## Docker
+
+Build the docker container with:
+
+```
+   docker build -t mod-kb-ebsco-java .
+```
+
+Test that it runs with:
+
+```
+   docker run -t -i -p 8081:8081 mod-kb-ebsco-java
+```
+
+## Installing the module
+
+Follow the guide of
+[Deploying Modules](https://github.com/folio-org/okapi/blob/master/doc/guide.md#example-1-deploying-and-using-a-simple-module)
+sections of the Okapi Guide and Reference, which describe the process in detail.
+
+First of all you need a running Okapi instance.
+(Note that [specifying](../README.md#setting-things-up) an explicit 'okapiurl' might be needed.)
+
+```
+   cd .../okapi
+   java -jar okapi-core/target/okapi-core-fat.jar dev
+```
+
+We need to declare the module to Okapi:
+
+```
+curl -w '\n' -X POST -D -   \
+   -H "Content-type: application/json"   \
+   -d @target/ModuleDescriptor.json \
+   http://localhost:9130/_/proxy/modules
+```
+
+That ModuleDescriptor tells Okapi what the module is called, what services it
+provides, and how to deploy it.
+
+## Deploying the module
+
+Next we need to deploy the module. There is a deployment descriptor in
+`target/DeploymentDescriptor.json`. It tells Okapi to start the module on 'localhost'.
+
+Deploy it via Okapi discovery:
+
+```
+curl -w '\n' -D - -s \
+  -X POST \
+  -H "Content-type: application/json" \
+  -d @target/DeploymentDescriptor.json  \
+  http://localhost:9130/_/discovery/modules
+```
+
+Then we need to enable the module for the tenant:
+
+```
+curl -w '\n' -X POST -D -   \
+    -H "Content-type: application/json"   \
+    -d @target/TenantModuleDescriptor.json \
+    http://localhost:9130/_/proxy/tenants/<tenant_name>/modules
+```
```

Comment:
Love this description - I think it will be very helpful for newbies to the project 👍